### ISSUE TITLE
image field: maximum files allowed 

### ISSUE THREAD

**Issue by tbba** [ryancramerdesign/ProcessWire#1023]
I had set an image field as container for a page with Max files allowed of 20.
I forgot about the limit later and did not have a hint left in descriptions.

Today I dropped 21 files onto the image field and wondered why 1 image did not upload.
Tried again and again... 

I think an alert would be good here that the maximum number of allowed images is reached here, instead of uploading, but then silently not saving it on page save.

**Comment by jacmaes** [comment]
+1

**Comment by ryancramerdesign** [comment]
When I exceed the max allowed files, I get a red error message in the field "Maximum files limit reached". It sounds like you are suggesting something it already does, so not sure I understand? Or maybe something about the field configuration differs from mine? To test, I created an images field and set the max at 3 files. I tried it once uploading 4 files together, and again uploading them individually, but same result: 3 files uploaded, then the 4th produced a red error message indicating the limit had been reached. Let me know if I'm missing something?
